tests/test_project/apps.py

```diff
@@ -15,9 +15,38 @@ class TestAppConfig(ApiAppConfig):
     def ready(self, *args, **kwargs):
         super().ready(*args, **kwargs)
         self.register_default_menu_items()
+        self.register_default_menu_groups()
 
     def register_default_menu_items(self):
         items = [{'model': 'test_project.Shelf'}]
         register_menu_items(items)
         # Required only for testing
         register_menu_items(items, name_menu='OPENWISP_TEST_ADMIN_MENU_ITEMS')
+
+    def register_default_menu_groups(self):
+        groups = [
+            {
+                'group_name': "List Group",
+                'group': [
```

Comment:
this could be `models` or `items` if we want something more generic (we can add support for linking pages that are not models so maybe it makes sense to use something generic)